src/ui/components/AppWrapper/AppWrapper.tsx

```diff
@@ -150,7 +149,8 @@ const AppWrapper = (props: { children: ReactNode }) => {
   }, []);
 
   useEffect(() => {
-    if (!isOffline && initialised) {
+    if (isOnline) {
+      dispatch(setPauseQueueIncomingRequest(true));
```

Comment:
Also. This says to pause the queue when we are online...? Won't it always be paused then? Please test the branch to make sure we can still receive notifications when logged in.